indigo_app/static/javascript/indigo/views/task_detail.js

```diff
@@ -30,6 +30,19 @@
     commentChanged: function(e) {
       var text = (e.target.value || '').trim();
       this.$('#task-comment-form #id_submit').attr('disabled', text.length === 0);
-    }
+
+      if (text.length > 0) {
+        this.$('#btn_request_changes').val('Comment and Request Changes');
```

Comment:
We use Sentence casing for buttons and headings.

```suggestion
        this.$('#btn_request_changes').val('Comment and request changes');
```